frontend/src/pages/line/LinePage.vue

```diff
@@ -52,11 +52,19 @@ export default {
   },
   async created() {
     // TODO 초기 역 데이터를 불러오는 API를 추가해주세요.
-    // const stations = await fetch("/api/stations")
-    // this.setStations([...stations])
-    // TODO 초기 노선 데이터를 불러오는 API를 추가해주세요.
-    // const lines = await fetch("/api/lines")
-    // this.setLines([...lines])
+    const response1 = await fetch("http://localhost:8080/stations")
+    if(!response1.ok){
+      throw new Error(`${response1.status}`);
+    }
```

Comment:
```suggestion
    if (!response1.ok) {
      throw new Error(`${response1.status}`);
    }
```

JS도 컨벤션 😉
`response1`, `response2`보다 더 괜찮은 이름 없을까요? 😢

Reply:
Member관련 JS 파일만 수정했습니다.
나머지 노선도관련 JS 파일은 3단계에서 기능 구현 완료후 적용하겠습니다. 👍